apmpackage/apm/0.1.0/manifest.yml

```diff
@@ -41,6 +41,11 @@ policy_templates:
         required: true
         show_user: true
         default: false
+      - name: kibana_api_key
+        type: string
```

Comment:
should this be `type: password`?

Reply:
it appears there is no such a thing...

Reply:
Am I looking at the wrong place here https://github.com/elastic/package-spec/blob/a8fe73d82b25806f6b96582b99993a63b67ef130/versions/1/data_stream/manifest.spec.yml#L53?